spec/controllers/api/teams_controller_spec.rb

```diff
@@ -369,7 +371,7 @@
       set_auth_headers
 
       team_params = { name: 'foo', private: true }
-      new_team = Team.create(users(:bob), team_params)
+      new_team = Team::Shared.create(users(:bob), team_params)
```

Comment:
```suggestion
      new_team = Team::Shared.create!(users(:bob), team_params)
```

check when to use bang! and when not ...